assignment7/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import Header from "./components/Header";
+import Search from "./components/Search";
+import PageNotFound from "./components/DeveloperCards/PageNotFound";
+import DeveloperCardsContainer from "./components/DeveloperCards/DeveloperCardsContainer";
+import Footer from "./components/Footer";
+import data from "./utils/data/data.json";
+import { Route, Routes } from "react-router-dom";
+import DeveloperDetails from "./components/DeveloperCards/DeveloperDetails";
+import FollowUp from "./components/DeveloperCards/FollowUp";
```

Comment:
Separate out your imports for better readability
```suggestion
// built-in imports
import React, { useState, useEffect } from "react";
import { Route, Routes } from "react-router-dom";

// components
import Header from "./components/Header";
import Search from "./components/Search";
import PageNotFound from "./components/DeveloperCards/PageNotFound";
import Footer from "./components/Footer";

import DeveloperCardsContainer from "./components/DeveloperCards/DeveloperCardsContainer";
import DeveloperDetails from "./components/DeveloperCards/DeveloperDetails";
import FollowUp from "./components/DeveloperCards/FollowUp";

import data from "./utils/data/data.json";
```